### PR TITLE
FIX: updated batch norm converter according onnx specs

### PR DESCRIPTION
**Pull request by ivkalgin**

### CODE REVIEW

File: onnx2torch/node_converters/batch_norm.py [hunk 1]
```diff
@@ -18,57 +23,77 @@
 }
 
 
+class OnnxBatchNorm(nn.Module, OnnxToTorchModule):  # pylint: disable=missing-docstring
+    def __init__(self, momentum: float = 0.1, epsilon: float = 1e-5):
```
Comment (senysenyseny16) [inline onnx2torch/node_converters/batch_norm.py:27]: Why are default arguments used here?

**Review by senysenyseny16 — APPROVED** [review]
